tokio/src/fs/mod.rs

```diff
@@ -86,6 +84,23 @@ pub use self::write::write;
 mod copy;
 pub use self::copy::copy;
 
+feature! {
+    #![unix]
+
+    mod symlink;
+    pub use self::symlink::symlink;
+}
+
+feature! {
+    #![windows]
+
+    mod symlink_dir;
+    pub use self::symlink_dir::symlink_dir;
+
+    mod symlink_file;
+    pub use self::symlink_file::symlink_file;
+}
```

Comment:
This will not show up on docs.rs at all. We should do something about that.

Reply:
It does if you pick windows? What happened before?

Reply:
It did not show up previously on the default view either, but I'm pretty sure rustdoc wont complain if we compile them on `windows or docsrs`, since it doesn't compile the body of the functions, only their signature.

Reply:
Interesting, I can give it a shot later.

Reply:
I gave it a shot, rustdoc did not like it due to broken intra doc links:

```
error: unresolved link to `std::os::windows::fs::symlink_dir`
  --> tokio/src/fs/symlink_dir.rs:13:12
   |
13 | /// [std]: std::os::windows::fs::symlink_dir
   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no item named `windows` in module `os`
   |
note: the lint level is defined here
  --> tokio/src/lib.rs:13:26
   |
13 | #![cfg_attr(docsrs, deny(broken_intra_doc_links))]
   |                          ^^^^^^^^^^^^^^^^^^^^^^

error: unresolved link to `std::os::windows::fs::symlink_file`
  --> tokio/src/fs/symlink_file.rs:13:12
   |
13 | /// [std]: std::os::windows::fs::symlink_file
   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no item named `windows` in module `os`

error: unresolved link to `std::os::windows::fs::symlink_dir`
  --> tokio/src/fs/symlink_dir.rs:13:12
   |
13 | /// [std]: std::os::windows::fs::symlink_dir
   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no item named `windows` in module `os`
```

The references fns do not exist in std under unix. This is non-standard, so I vote we skip unless you have another strategy to try?

Reply:
Hm. Annoying. But yes, let's not do it for now.